src/sass/page-grid/page-grid.scss

```
@@ -15,7 +15,7 @@
         var(--page-grid-max-width) + (var(--page-grid-outside-margins) * 2)
     );
 
-    @media screen and (min-width: $_screen-size-SM) {
+    @media all and (min-width: $_screen-size-SM) {
```

Isn't `all and` redundant here? This could be simplified to
```suggestion
    @media (min-width: $_screen-size-SM) {
```

---

Yes, I'd forgotten the omission implied `all`. I'll update. Thanks!